README.md

```diff
@@ -50,6 +50,12 @@ There are methods for the four canonical event types: `created`, `updated`,
 the publisher floods with `noop`s for all existing entities so subscribers can
 refresh their view of the domain.
 
+A timestamp argument may be passed (it will be set by the bus automatically
+otherwise); it must be an integer number of milliseconds since the UNIX Epoch:
+
+```ruby
+client.created('widgets', 'https://app.example.com/widgets/1', 1473080555409)
```

Comment:
> I suppose this was added in another pull-request?

No, this is just documenting something that was already there.

> I'm wondering whether that could cause issues when debugging. For instance, if any arbitrary time can be passed in the message what would be a good source of truth when looking at time-sensitive events?

That's the point — it's up to the event sender. If it's sending events "as they happen" (the general case), letting Routemaster set the timestamp as the reception time is good enough.
If it's batching events, or sending them form Resque jobs for instance, then the publisher should set timestamps in the "past".

In any case, clients that need authoritative timestamp info are probably best off querying the resource for an `updated_at`.

One future improvement could be for Routemaster to prevent _future_ timestamps to be set (either publisher- or bus-side).